opm-core/Backups.rst

```diff
@@ -0,0 +1,22 @@
+Backups
+=======
+
+Usual backup procedures apply. The procedure should include the way to reinstall the extensions on a virgin server.
+
+Restoring logical backups
+-------------------------
+
+In a general way, restoring a backup created with pg_dump should not be a problem as the backup contains the necessary `CREATE EXTENSION` orders. The extension binaries must be present.
+
+You may wish to parallelize the restoration to spare time. In this case, it may fail with a foreign key error, as the extensions install tables with foreign keys at the very beginning of the restore, and pg_restore does not try to respect this order. The trick is to parallelize only the restoration of the biggest tables.
+
+The following commands assume that the database and the roles were already created :
```

Comment:
It also assumes that non-plain format was used.  It's probably obvious but it wouldn't hurt to mention it, and probably give an example of how to perform the logicial backup.

Reply:
Done.